src/elements/content-sidebar/activity-feed/_mixins.scss

```diff
@@ -0,0 +1,6 @@
+$bcs-ActivityFeedItem-padding-vertical: 20px;
+$bcs-ActivityFeedItem-padding-horizontal: 25px;
+
+@mixin bcs-ActivityFeedItem {
```

Comment:
Would this be `bcs-ActivityFeedCard`, since we can have feed items, such as versions, that have a different visual treatment? Instead of a mixin, could we replace the plain `div` elements of each activity type with an ActivityCard component?